internal/ir/xds.go

```diff
@@ -2398,6 +2398,16 @@ type CircuitBreaker struct {
 
 	// The maximum number of parallel retries that Envoy will make.
 	MaxParallelRetries *uint32 `json:"maxParallelRetries,omitempty" yaml:"maxParallelRetries,omitempty"`
+
+	// PerHost defines per-host Circuit Breakers
+	PerHost *PerHostCircuitBreakers `json:"perHost,omitempty"`
+}
+
+// PerHostCircuitBreakers defines the per-host Circuit Breaker configuration.
+// +k8s:deepcopy-gen=true
+type PerHostCircuitBreakers struct {
+	// MaxConnections configures the maximum number of connections that Envoy will establish per-host to the referenced backend defined within a xRoute rule.
+	MaxConnections *uint32 `json:"maxConnections,omitempty"`
```

Comment:
nit? there's a default value in API, so I think we can use non point here, will that make thing easier?

Reply:
Sure, should that also be changed for the non-PerHost CircuitBreaker field? I was aiming to align with the existing fields

Reply:
we can do the refactor later.